source/includes/fact-write-blocking-enable.rst

```diff
@@ -0,0 +1,8 @@
+When write-blocking is enabled, ``mongosync`` blocks writes:
+
+- On the destination cluster during replication
```

Comment:
We prefer to use "synchronization" instead of "replication" when talking about mongosync/C2C to avoid confusion with server replication.

Reply:
Done - per PM, the docs typically use "sync" rather than "synchronization"

Reply:
sounds good!